Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -47,6 +47,10 @@ set_ruby_variables() {
     if [[ -n "${HOMEBREW_ARTIFACT_DOMAIN}" ]]
     then
       ruby_URLs+=("${HOMEBREW_ARTIFACT_DOMAIN}/v2/homebrew/portable-ruby/portable-ruby/blobs/sha256:${ruby_SHA}")
+      if [[ -n "${HOMEBREW_ARTIFACT_DOMAIN_NO_FALLBACK}" ]]
+      then
```

Comment:
```suggestion
      then
        ruby_URL="${ruby_URLs[0]}"
```
Not really sure if it's needed though.

Reply:
Yeh, good catch. Might rework to avoid the early return here instead 🤔